src/mineflayer/events/OnceLogin.mjs

```diff
@@ -44,6 +44,30 @@ export default {
         process.exit(1);
       }, 1000);
     }
+    if (
+      bot.utils.getPermissionsByUser({ name: bot.bot.username }) <
+      Permissions.BotAccount
+    ) {
+      let data = bot.utils.playerNamesDatabase.get("data");
+      let userObject = data.find((x) =>
+        x.accounts.find((y) => y.name === bot.bot.username),
+      );
+      if (!userObject)
+        userObject = {
+          accounts: [
+            {
+              name: bot.bot.username,
+              uuid: await bot.utils.getUUID(bot.bot.username),
+            },
+          ],
+          preferredName: bot.bot.username,
+        };
+      userObject.permissionRank = Permissions.BotAccount;
+      if(data[data.indexOf(userObject)]) data[data.indexOf(userObject)] = userObject
```

Comment:
No prettier, somehow, but I'll let it slide

Reply:
Used prettier on file